### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jordanfrazier97**

### CODE REVIEW

**Comment by whanley** [comment]
Good start @jordanfrazier97 -- make sure that you fill in all of the information so that it matches your issue.

File: 1906-05-28.xml [hunk 1]
```diff
@@ -0,0 +1,1946 @@
...
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <!-- Enter issue number, date, and correct number of pages -->
+                    <head type="sub">No. <measure quantity="7504">7,504</measure>] ALEXANDRIA, THURSDAY, JULY 13, 1905. [SIX PAGES P.T.
```
Comment (znr13) [inline 1906-05-28.xml]: Correct this.

**Comment by znr13** [comment]
@jordanfrazier97 , keep at it. Try to get your page 3 complete quickly.

**Comment by whanley** [comment]
@jordanfrazier97 keep at it--looking for page 3.

**Pull request merged**